cmd/analyzer/parse_args.go

```diff
@@ -77,7 +111,12 @@ func ParseInArgs(cmdlineArgs []string) (*InArgs, error) {
 	args.Debug = flagset.Bool("debug", false, "run in debug mode")
 	args.Version = flagset.Bool("version", false, "prints the release version number")
 
-	err := flagset.Parse(cmdlineArgs)
+	err := parseCmdLine(cmdlineArgs)
+	if err != nil {
+		return nil, err
+	}
+
+	err = flagset.Parse(cmdlineArgs)
```

Comment:
add some comment about why we need to call `parseCmdLine` prior to `flagset.Parse`, explain shortly what is added in this processing of the cli args